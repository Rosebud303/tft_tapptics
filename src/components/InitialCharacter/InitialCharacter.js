import React from 'react'

const InitialCharacter = () => {
  return (
    <div>
      
    </div>
  )
}

export default InitialCharacter;
