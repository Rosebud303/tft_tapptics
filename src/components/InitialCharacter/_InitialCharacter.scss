.images {
  width: 15%;
  height: 7rem;
}

.car-imgs {
  width: 4rem;
  height: 100%;
  transition: 1.5s;
  opacity: 0.7;
  filter: drop-shadow(2px 2px 9px #673AB7) drop-shadow(-2px -2px 17px white);
  &:hover {
    filter: drop-shadow(2px 2px 9px white) drop-shadow(-2px -2px 17px red);
    cursor: pointer;
    opacity: 1;
  }
}
