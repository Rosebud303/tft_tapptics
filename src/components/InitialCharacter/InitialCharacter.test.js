import React from 'react';
import { shallow } from 'enzyme';
import InitialCharacter from './InitialCharacter';

describe('Champions', () => {

  let wrapper;
  let mockName;
  let mockImg;
  let mockSyns;
  let mockRare;

  beforeEach(() => {
    mockName = 'Bob';
    mockImg = 'img.png';
    mockSyns = ['knight', 'warrior']
    mockRare = 2
    wrapper = shallow(<InitialCharacter name={mockName}
                                        img={mockImg}
                                        synergies={mockSyns} 
                                        rarity={mockRare}/>);
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});