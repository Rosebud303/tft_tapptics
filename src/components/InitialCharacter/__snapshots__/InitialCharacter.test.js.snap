// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Champions should match the snapshot with all the data passed in 1`] = `
<div
  className="images"
>
  <a
    data-tip={true}
  >
    <img
      className="car-imgs"
    />
  </a>
  <ReactTooltip
    clickable={false}
    insecure={true}
    resizeHide={true}
    type="dark"
    wrapper="div"
  >
    <li />
    <li />
  </ReactTooltip>
</div>
`;
